lib/assets/javascripts/utils/notificationHelper.js

```diff
@@ -1,32 +1,58 @@
 import { isString, isObject } from './isType';
+
 /*
   Helpers that will display the specified message in in the notification
   area at the top of the page
 */
-export const renderNotice = (msg) => {
-  const notificationArea = $('#notification-area');
 
-  if (isString(msg) && isObject(notificationArea)) {
-    notificationArea.removeClass('alert-warning').addClass('alert-info');
-    notificationArea.find('i, span').remove();
-    notificationArea.append(`
-      <i class="fa fa-check-circle" aria-hidden="true"></i><span>${msg}</span>`);
-    notificationArea.removeClass('hide');
-  }
-};
+export function hideNotifications() {
+  $('#notification-area')
+    .addClass('hide')
+    .removeClass('notification-area--floating');
+}
 
-export const renderAlert = (msg) => {
+function renderMessage(options = {}) {
   const notificationArea = $('#notification-area');
 
-  if (isString(msg) && isObject(notificationArea)) {
-    notificationArea.removeClass('alert-info').addClass('alert-warning');
+  if (isString(options.message) && isObject(notificationArea)) {
+    notificationArea
+      .removeClass('alert-info', 'alert-warning')
+      .addClass(options.className);
+
+    if (options.floating) {
+      notificationArea.addClass('notification-area--floating');
+    }
+
```

Comment:
Reusing the `#notification-area` like this may cause issues with any global notifications that are active. If so it might be better to either pass in the `notificationArea` so that the caller can determine which DOM element they want to work with.

Reply:
This is how the original JS code was set to function. But I agree, it's not ideal. 

If you like, I can refactor this on a separate Issue?

Reply:
Ok. hold off on refactoring for now.